src/br/com/ciandt/jwillian/service/ComentarioService.java

```diff
@@ -0,0 +1,41 @@
+package br.com.ciandt.jwillian.service;
+
+/**
+ * Classe de servi�os para coment�rios
+ * @author jwillian
+ */
+public class ComentarioService {
+
+	//Metodo de cria��o da constante "COMENT�RIO_REMOVIDO" tipo String
+	private final static String COMENTARIO_REMOVIDO = new String("Coment�rio removido!"); 
+
+
+	/**
+	 * Metodo contrutor da classe
+	 */
+	public ComentarioService() {}
+	
+	/**
+	 * Metodo que verifica o comentario
+	 * @param comentario {@link String} coment�rio
+	 * @return true caso valor diferente de null ou false se for null
+	 */
+	public Boolean verificarComentario (String comentario) 
+	{
+		//if que verifica se o valor do coment�rio � True ou False
+		if(comentario != null){
```

Comment:
no if seguir esse padrão. atentar-se aos espaços
```suggestion
		if (comentario != null) {
```